github-activity-tracker/backend/.env.example

```
@@ -3,4 +3,12 @@ RDS_PORT=5432
 RDS_DATABASE=github_data
 RDS_USER=your-rds-user
 RDS_PASSWORD=your-rds-password
-GITHUB_TOKEN=your-github-token
\ No newline at end of file
+GITHUB_TOKEN=your-github-token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_TOKEN key should go before the RDS_DATABASE key

(UnorderedKey)